src/Simplify.elm

```diff
@@ -6607,19 +6607,14 @@ collectionMapChecks :
     -> List (Error {})
 collectionMapChecks collection checkInfo =
     firstThatReportsError
-        [ \() ->
-            mapIdentityChecks
-                { moduleName = AstHelpers.moduleNameFromString collection.moduleName
-                , represents = collection.represents
-                }
-                checkInfo
+        [ \() -> mapIdentityChecks collection checkInfo
         , \() ->
             case secondArg checkInfo of
                 Just collectionArg ->
                     if collection.isEmpty checkInfo.lookupTable collectionArg then
                         [ Rule.errorWithFix
                             -- TODO rework error info
-                            { message = "Using " ++ collection.moduleName ++ ".map on " ++ collection.emptyDescription ++ " will result in " ++ collection.emptyDescription
+                            { message = "Using " ++ qualifiedToString ( collection.moduleName, "map" ) ++ " on " ++ collection.emptyDescription ++ " will result in " ++ collection.emptyDescription
```

Comment:
Not convinced about `qualifiedToString` here either :thinking:  (but happy to be convinced)

Reply:
Just to be sure: Do you know the difference between `qualifiedToString` and `qualifiedToString (qualify ...)`?
- just `qualifiedToString` always prints with the exact module name you provide (usually fully qualified), even when for example exposed in the imports
- `qualify` takes a fully qualified thing and adapts the lowest currently allowed qualification. E.g. it sets the module name to `[]` when exposed in the current module

If you did know that and asking why I'm using `qualifiedToString` instead of a raw string, that's just to make it easier in the future to generalize the error and fix format. I'd like to do that in the next PR so this change is in a bit of a limbo